lib/matplotlib/text.py

```diff
@@ -239,10 +240,16 @@ def update(self, kwargs):
         """
         Update properties from a dictionary.
         """
-        bbox = kwargs.pop('bbox', None)
```

Comment:
I prefer this way because a) smaller change b) does not bring in OrderedDict  c) does not provide an undocumented but abusable API surface dependent on an 'isinstance' check.